manifests/database/postgresql.pp

```diff
@@ -70,10 +70,22 @@
   }
 
   if $manage_database {
+    # TODO: Workaround, until "postgresql" accepts Sensitive
```

Comment:
It has been released in puppetlabs-postgresql 7.5.0 thanks to your work on the topic, thanks!
https://github.com/puppetlabs/puppetlabs-postgresql/pull/1279

This will need a bump of the minimal module version in `metadata.conf`.

Reply:
I removed the Workarounds, and adapted the Versions in `metadata.json`.